types/size.pp

```diff
@@ -1,6 +1,6 @@
 type Bacula::Size = Variant[
   Integer,
   Pattern[
-    /(?i:\A\d+[mg]b?\Z)/,
+    /(?i:\A\d+\s*[mg]b?\Z)/,
```

Comment:
:+1: bacula strips spaces before parsing lines

Reply:
I'm glad this tripped me TBH.  I knew custom types were possible, but hadn't realized just how cleanly and simply this could be done now.  I have lots of cases in other modules I maintain where my typing causes nasty line wrapping and just an ugly mess in class/def params.  So happy I am.